interfaces/apparmor/spec.go

```diff
@@ -96,6 +101,15 @@ func (spec *Specification) setScope(securityTags []string) (restore func()) {
 	}
 }
 
+func (spec *Specification) Features() []string {
+	return spec.features
+}
+
+func (spec *Specification) MockSetFeatures(features []string) {
```

Comment:
this probably can just be called "MockFeatures" no? I don't think we use the word combination "mock set" anywhere so far

Reply:
done